src/modules/jackrack/CMakeLists.txt

```diff
@@ -40,6 +45,13 @@ if(GPL AND TARGET PkgConfig::xml AND TARGET PkgConfig::glib AND ladspa_h_FOUND)
     target_sources(mltjackrack PRIVATE filter_jackrack.c)
     install(FILES filter_jackrack.yml DESTINATION ${MLT_INSTALL_DATA_DIR}/jackrack)
   endif()
+
+  if(USE_LV2 AND TARGET PkgConfig::lilv)
+    target_link_libraries(mltjackrack PRIVATE PkgConfig::lilv)
+    target_sources(mltjackrack PRIVATE filter_lv2.c producer_lv2.c lv2_context.c lv2_plugin.c lv2_process.c lv2_plugin_settings.c)
```

Comment:
Please add the new header files to this as well as my IDE uses this to show the project files. You can see all of our CMakeLists.txt do that including other `target_sources` in the same file.